--- a/config/m4/hdf5_f90.m4
+++ b/config/m4/hdf5_f90.m4
@@ -135,18 +135,18 @@
     AC_MSG_CHECKING([for HDF5 using h5pfc/h5fc system compilers]) ;
     #
     h5pfc="none"
-    if test -e $with_hdf5_path/bin/h5pfc; then 
-       h5pfc=$with_hdf5_path/bin/h5pfc; 
-     elif command -v h5pfc > /dev/null; then 
-       h5pfc=$(command -v h5pfc)  
-    fi
+#    if test -e $with_hdf5_path/bin/h5pfc; then 
+#       h5pfc=$with_hdf5_path/bin/h5pfc; 
+#     elif command -v h5pfc > /dev/null; then 
+#       h5pfc=$(command -v h5pfc)  
+#    fi
     # 
     h5fc="none"
-    if test -e $with_hdf5_path/bin/h5fc ; then 
-       h5fc=$with_hdf5_path/bin/h5fc; 
-    elif command -v h5fc > /dev/null; then 
-       h5fc=$(command -v h5fc) 
-    fi
+#    if test -e $with_hdf5_path/bin/h5fc ; then 
+#       h5fc=$with_hdf5_path/bin/h5fc; 
+#    elif command -v h5fc > /dev/null; then 
+#       h5fc=$(command -v h5fc) 
+#    fi
     #
     if test -e $h5pfc; then
        try_HDF5_LIBS=`$h5pfc -show | awk -F'-L' '{@S|@1=""; for (i=2; i<=NF;i++) @S|@i="-L"@S|@i; print @S|@0}'`
