--- a/lib/archive/package.list
+++ b/lib/archive/package.list
@@ -53,7 +53,7 @@
 url_netcdff=https://github.com/yambo-code/yambo/files/855477/$(tarball_netcdff)
 url_hdf5=https://github.com/yambo-code/yambo/files/838459/$(tarball_hdf5)
 url_libxc=https://github.com/yambo-code/yambo/files/783150/$(tarball_libxc)
-url_iotk=https://github.com/yambo-code/yambo/files/783147/$(tarball_iotk)
+url_iotk=https://github.com/yambo-code/yambo/files/962173/$(tarball_iotk)
 url_etsf_io=https://github.com/yambo-code/yambo/files/845218/$(tarball_etsf)
 url_lapack=https://github.com/yambo-code/yambo/files/783151/$(tarball_lapack)
 url_blacs=https://github.com/yambo-code/yambo/files/783152/$(tarball_blacs)
