--- a/config/mk/global/functions/get_libraries.mk
+++ b/config/mk/global/functions/get_libraries.mk
@@ -3,8 +3,8 @@ define get_external_libraries
  mkdir -p lib/archive; \
  cd lib/archive; \
  if [ "$$LIB2DO" = "all" ] ; then \
-  $(MAKE) $(MAKEFLAGS) -f Makefile.loc all ; \
+  $(MAKE) -f Makefile.loc $(MAKEFLAGS) all ; \
  else \
-  $(MAKE) $(MAKEFLAGS) -f Makefile.loc $$LIB2DO ; \
+  $(MAKE) -f Makefile.loc $(MAKEFLAGS) $$LIB2DO ; \
  fi
 endef
