--- a/configure
+++ b/configure
@@ -9898,7 +9898,7 @@
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking the yambo-libraries git repository" >&5
 $as_echo_n "checking the yambo-libraries git repository... " >&6; }
   if ! test -d "yambo/driver/src"; then
-    git clone git@github.com:yambo-code/yambo-libraries.git yambo >& /dev/null
+    git clone https://github.com/yambo-code/yambo-libraries.git yambo >& /dev/null
   else
     cd yambo
     git checkout master >& /dev/null
